src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain;
+
+public class Car {
+
+    private static final int INCREASE_STANDARD_NUMBER = 4;
+    private int moveCount = 0;
```

Comment:
상수와 멤버변수 간 구분을 위해 개행을 넣어주는 것이 좋습니다.
```suggestion
    private static final int INCREASE_STANDARD_NUMBER = 4;

    private int moveCount = 0;
```